### ISSUE TITLE
What the Xsub, Xview and 1-shot

### ISSUE THREAD

**Issue by leviethung2103** [Walter0807/MotionBERT#74]
Hello,

I am starting to fine tuning the MB. However, I am confused with these terms: `Xsub`, `Xview` and `1-shot`. Especially the term 1-shot.

In my scenario, I have one camera at the fixed location. How can fine-tune the MB ? Furthermore, my dataset only contains 2 actions (2 classes)

The option should I choose is Cross View, Cross Subject or 1-shot. Thank you 

https://github.com/Walter0807/MotionBERT/blob/main/docs/action.md

**Comment by Walter0807** [comment]
X-sub or X-view are ways to split your train/test set, which you can  decide by yourself on your own dataset and no need to follow the practices on NTU-RGB+D.

The oneshot setting would be suitable if you have extremely few labeled data. Otherwise fine tuning on your dataset might be better.
